modules/mediawiki/manifests/init.pp

```diff
@@ -216,4 +216,11 @@
         backup  => false,
         recurse => 1,
     }
+    exec { 'create python venv':
+        command => '/usr/bin/python3 -m venv /srv/pythonenv && /srv/pythonenv/bin/pip3 install Miraheze-PyUtils',
+        require => Package[python3],
```

Comment:
Untested but I am guessing this will fail since Python3 isn't actually a package resource managed by Puppet.

Reply:
Not sure on that. We can make it so.